crates_contrib/utils.py

```diff
@@ -898,7 +898,9 @@ def _validate_coordsys(self, cname):
     def _convert(self, direction, mappings, x, y):
         """Convert a single (x,y) using the direction and mappings."""
 
-        pt = [[x, y]]
+        # Need to make sure x and y are float to avoid Transform bug
+        # when inputs are integer (forces output to be integer)
+        pt = [[x * 1.0, y * 1.0]]
```

Comment:
I could argue that "send in an int, get an int back" makes sense, but probably not in Python...

Reply:
Not when going from logical to celestial.